### PR TITLE
Bringing in MACE-MP workflows to CARMM

### PR DESCRIPTION
**Pull request by ikowalec**
Now removed the [WIP] tag.

TODOs: 
- minimise code repeats and merge `react.py` and `react_mace.py`
- vibrations
- MD (missing for FHI-aims also)

Done:
  - geometry optimisation with restarts
  - NEBs with restart
  - Added MACE to the CI test
  - choice of MACE_MP, MACE_OFF and MACE_ANICC models added

### CODE REVIEW

File: examples/run_workflows_ReactMACE.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Author: Igor Kowalec
+'''
+This example shows how to use the work flow functionality with CARMM to optimise
+reactants/products and find transition states in an automated manner using the MACE-MP force field.
+'''
+
+
+def test_run_workflows_ReactAims():
```
Comment (GabrielBram) [inline examples/run_workflows_ReactMACE.py:8]: Small error - still retains name of old ReactAims.

**Review by GabrielBram — CHANGES_REQUESTED** [review]
Thank you very much for this, Igor. I can contribute to extending the transition state search in the coming days. I have tested ML-NEB/AutoNEB/standard ASE NEB, and I can confirm that the training time needed for ML-NEB far exceeds the savings.

**Comment by ikowalec** [comment]
> Look through the MACE ASE calculator, they unfortunately import torch at the head so I don't believe there's a way around this. Would this require Andy to update the CI test runner?

We would have to cache the pytorch installation somehow - we can't just download and rebuild 2.5gb of packages on each pull request. Something to think about.

**Comment by ikowalec** [comment]
> Look through the MACE ASE calculator, they unfortunately import torch at the head so I don't believe there's a way around this. Would this require Andy to update the CI test runner?

now resolved by amending the `main.yml`

File: examples/run_workflows_ReactMACE.py (hunk 1 above)
Comment (ikowalec) [reply examples/run_workflows_ReactMACE.py]: > Small error - still retains name of old ReactAims.

now resolved

**Review by GabrielBram — APPROVED** [review]

**Pull request merged**